transcom/mymove
# B 18061 int

## [B-18061 ](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3a870623)

## Summary - AOA Packet Combination

Is there anything you would like reviewers to give additional scrutiny?
PDFs with bookmarks should be processed correctly, but this was an issue during development.

[this article](tbd) explains more about the approach used.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Run Server, Server Tests
2. Login as a Client User
3. Check Swagger for the ppm -> ppm-shipments/{ppmshipmentid}/aoa-packet api endpoint
4. Find a suitable PPMShipmentId to test with from database 
5. Insert the PPMShipmentId into the "Try it out" section
6. Test the API, then grab the example URL
7. Paste the example URL into your browser
8. Ensure PDF with SSW and Orders Document appears
9. Repeat steps 2-8 as an office user, with the office API endpoint under the same category and name

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
